gradio/cli/commands/components/files/README

```diff
@@ -0,0 +1,11 @@
+---
+tags: [gradio-custom-component, gradio-custom-component-template-{{ template }}]
+title: {theme_name}
+colorFrom: orange
+colorTo: purple
+sdk: gradio
```

Comment:
Confused about this -- why is the sdk gradio if this is a Docker Space? I'm probably missing something

Reply:
Is this file not the readme in the uploaded Space?

Reply:
Yea sorry I need to delete this file! It's not used right now.